crates/bevy_input_focus/src/tab_navigation.rs

```diff
@@ -378,23 +376,25 @@ mod tests {
         let tab_entity_1 = world.spawn((TabIndex(0), ChildOf(tab_group_entity))).id();
         let tab_entity_2 = world.spawn((TabIndex(1), ChildOf(tab_group_entity))).id();
 
-        let mut system_state: SystemState<TabNavigation> = SystemState::new(world);
-        let tab_navigation = system_state.get(world);
-        assert_eq!(tab_navigation.tabgroup_query.iter().count(), 1);
-        assert_eq!(tab_navigation.tabindex_query.iter().count(), 2);
+        let system = move |tab_navigation: TabNavigation| {
+            assert_eq!(tab_navigation.tabgroup_query.iter().count(), 1);
+            assert_eq!(tab_navigation.tabindex_query.iter().count(), 2);
```

Comment:
Ignore my change please. I don't know how to fix this test. The tabindex_query is now matching bunch of other entities since queries are entities now.

Reply:
I think removing the assertion would be OK. The tabindex_query is trying to recursively locate all descendants of the tabgroup entity which have a tabindex component, so it's not just a simple child query. The number of of entities queried isn't important, what is important is the number of tabindex entities found, but the assertion isn't testing that.